astronify/series.py

```diff
@@ -60,9 +61,22 @@ def data(self):
         return self._data
 
     @data.setter
-    def data(self, value):
-        assert isinstance(value, Table), 'Data must be a Table.'
-        self._data = value
+    def data(self, data_table):
+        assert isinstance(data_table, Table), 'Data must be a Table.'
+
+        # Removing any masked values as they interfere with the sonification
+        if isinstance(data_table[self.val_col], MaskedColumn):
+            data_table = data_table[~data_table[self.val_col].mask]
+        if isinstance(data_table[self.time_col], MaskedColumn):
+            data_table = data_table[~data_table[self.time_col].mask]
+
+        # making sure we have a float column for time
+        if isinstance(data_table[self.time_col], Time):
+            float_col = f"{self.time_col}_jd"
```

Comment:
This is OK for now, we just need to make sure that if we ever use the Time column information anywhere, that for Kepler, K2 and TESS, the timestamps need to have the offsets added back in so the astropy JD conversion results in correct Julian Dates.

Reply:
I believe that all of that should be done prior to turning the column into a Time object. I don't think it's in the purview of this code to check if Time objects are correct. 

If this is a serious concern I will just change the column name to time_col_float so it doesn't imply anything about the time scale.